md/get-bonita-provided-applications.md

```diff
@@ -0,0 +1,34 @@
+# Deploying the Bonita User and Administrator Applications
+
+This page describes how to get the two applications and how to deploy them into a Tomcat bundle or the cloud.
+
+## How to get the applications 
+
+You can get the applications in two different ways, either from the Studio welcome page or from their two GitHub repositories.
```

Comment:
I would remove all the following content. I've added what was missing (deploying without BCD) in the overview pages of the 2 Bonita applications.